.github/workflows/deploy-dev.yml

```diff
@@ -0,0 +1,25 @@
+name: Deploy to Dokku Dev
+
+on:
+#  pull_request:
+  push:
+    paths-ignore:
+      - '**/*.md'
+      - 'docs/**'
+      - '.gitignore'
+  workflow_dispatch:
+jobs:
```

Comment:
It would be good to add a blank line between `on` and `jobs`. Also, it is better to add a blank line between steps for more readability.

Reply:
Done!